src/js/components/state/StateLoading.jsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * StateLoading.jsx
+ * Created by Lizzie Salita 5/2/18
+ */
+
+import React from 'react';
+
+import Error from 'components/sharedComponents/Error';
+
+export default class StateLoading extends React.Component {
```

Comment:
Do we need this component if it just returns a single hard-coded `Error` component?

Reply:
Updated to use shared `Error` component and removed `StateLoading` and `StateError` components.